rex/sam_resource.py

```diff
@@ -33,6 +37,8 @@ class SAMResource:
                 'lineardirectsteam': ('dni', 'dhi', 'wind_speed',
                                       'air_temperature', 'dew_point',
                                       'surface_pressure'),
+                'wind': ('pressure', 'temperature', 'winddirection',
```

Comment:
I thought we decided to drop pv and wind and only use the SAM tech names?

Reply:
I thought in rex we were going to do "all of the above" to keep this repo more agnostic?

Reply:
Sounds good to me